README.md

```diff
@@ -120,8 +121,10 @@ optional arguments:
                         `--template-variables-path=namespace=path_or_url`.
                         Namespaces must match the regex `[a-zA-
                         Z_][a-zA-Z0-9_]*`.
-  --skip-citations      Skip citation and reference processing. Specify when
-                        using the pandoc-manubot-cite filter. If citation-
+  --skip-citations      Skip citation and reference processing. Support for
```

Comment:
Are you keeping this as a required argument to help differentiate calls to the old and current `manubot process`?

Reply:
Yes, the idea is if you are using the old behavior of `manubot process` without `--skip-citations` you will get an error. So this is a way to ensure users are aware of the change and upgrade their workflow.